python/VERSION

```diff
@@ -1 +1 @@
-0.14.0
+0.15.0rc0
```

Comment:
is this still rc0? looks upstream use `0.15.0`

Reply:
I think they the commit that updates it will be in one of the next batches.

This is just 15 commits from the oldest to newer.

Reply:
ah ok